wrapper/tl.js

```diff
@@ -0,0 +1,6 @@
+const tljs = require('@y21/tljs');
+tljs.initializeWasmSync();
+
+module.exports = function(html, callback) {
+    tljs.parse(html).then(() => callback());
```

Comment:
Please relay any potential error to the callback. You could use `util.callbackify`.